IDE/Espressif/ESP-IDF/examples/wolfssl_server/main/server-tls.c

```diff
@@ -42,7 +42,87 @@
     #include <wolfssl/wolfcrypt/mem_track.h>
 #endif
 
-const char *TAG = "tls_server";
+#if defined(WOLFSSL_ESPWROOM32SE)
+static const char* TAG = "tls_server";
+#else
+const char* TAG = "tls_server";
+#endif
+
+static void ShowCiphers(void)
+{
+    static char ciphers[4096];
```

Comment:
This should not be static. If its static the compiler will reserve 4096 bytes globally all the time for this function.

Reply:
Fixed. And only enabled the function when DEBUG_WOLFSSL is enabled.